File: 2011-final-value-of-variable-after-performing-operations/2011-final-value-of-variable-after-performing-operations.java
class Solution {
    public int finalValueAfterOperations(String[] operations) {
        Map<String, Integer> store = new HashMap<>();

        store.put("++X", 1);
        store.put("X++", 1);
        store.put("--X", -1);
        store.put("X--", -1);

        int sum = 0;

        for (String s : operations) sum += store.get(s);

        return sum;
    }
}

File: 2011-final-value-of-variable-after-performing-operations/2011-final-value-of-variable-after-performing-operations.py
class Solution:
    def finalValueAfterOperations(self, operations: List[str]) -> int:
        ops = {
            "++X": 1,
            "X++": 1,
            "--X": -1,
            "X--": -1,
        }

        total = 0
        for op in operations:
            total += ops[op]

        return total
